actix-web/src/sec-5-errors.md

```diff
@@ -259,6 +260,13 @@ fn index(_req: HttpRequest) -> Result<&'static str, UserError> {
 # }
 ```
 
-[errorhelpers]: https://actix.rs/actix-web/actix_web/error/index.html#functions
+By dividing errors into those which are user facing and those which are not, we
+can ensure that we don't accidentally expose users to errors thrown by
+application internals which they weren't meant to see.
```

Comment:
I think this section flows a little better with a follow up paragraph that restates the thesis.